controller/src/main/java/io/pravega/controller/eventProcessor/impl/EventProcessorCell.java

```diff
@@ -168,6 +169,7 @@ private void restart(Throwable error, T event) {
 
         private void handleException(Exception e) {
             ExceptionHandler.Directive directive = eventProcessorConfig.getExceptionHandler().run(e);
+            log.info("Exception handler directive is {}", directive);
```

Comment:
here as well, this should be a `warn` log

Reply:
This log is to print the value of **directive**, so i think having it info level is much better.